controllers/home.js

```diff
@@ -4,11 +4,11 @@
  */
 
 exports.index = function(req, res) {
-  if (req.user) {
-    res.redirect('/learn-to-code')
-  } else {
+if (reg.user && process.env.NODE_ENV !== 'production'){ //added !== dev because crash
+	res.redirect('/learn-to-code')
```

Comment:
If you clone the repo and try to run the site if you attempt to access challenges or bonefire it crashes the server. I discussed with nathan and this was the agreed solution.